Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/EnumMemberDeclaration.cs

```diff
@@ -6,6 +6,17 @@ public static Doc Print(EnumMemberDeclarationSyntax node)
     {
         var docs = new List<Doc>
         {
+            node.GetLeadingTrivia()
+            .Any(
```

Comment:
On an unrelated note, you reckon there should be an indent here? Since it's the start of a method chain?